Hypnotic-Herons/Sprint-2
# added computer template and added many to many references on employee…

## Description of Proposed Changes
- added template for computer
- added many to many relationships for employee (computer and trainingprograms)

## Steps to Test

Outline the steps to test

```
git fetch --all
git checkout computerView
code .

```

1. python manage.py makemigrations HRIT
1. python manage.py migrate
1. python manage.py createsuperuser (this is just for your local use)
1. login on the admin view http://127.0.0.1:8000/admin
1. add a department
1. add a computer
1. add a few employees (don't worry about training programs)
1. look at the browser view for employees to see if they are there http://127.0.0.1:8000/HRIT/employees/
1. look at the browser view for computers to see if they are there http://127.0.0.1:8000/HRIT/computers/
1. you should see the employees and the computers you just entered

  
## Impacted Areas in Application

Database for employees table/View for employees
View for computers


## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
